.travis.yml

```diff
@@ -11,12 +11,14 @@ env:
   - WAGTAIL_VERSION=2.2.2
   - WAGTAIL_VERSION=2.3
   - WAGTAIL_VERSION=2.4
-  - WAGTAIL_VERSION=2.5.1
+  - WAGTAIL_VERSION=2.5.2
+  - WAGTAIL_VERSION=2.6.3
+  - WAGTAIL_VERSION=2.7.3
+  - WAGTAIL_VERSION=2.8.2
+  - WAGTAIL_VERSION=2.9.2
```

Comment:
Is het nodig om ze ver terug te testen? Volgens mij gaat Wagtail zelf niet zo ver in support.

Misschien ook handig om versie specs te gebruiken? (`WAGTAIL_VERSION='>=2.5,<2.6'`, en dan later `pip install wagtail$WAGTAIL_VERSION`)